wp-content/themes/haruoshi_origin/single-works.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * This is single-works.php
+ *
+ * @package haruoshi-origin
+ */
+
+get_header(); ?>
+
+<article class="container">
+	<h1 class="container__title"><?php the_title(); ?></h1>
+	<p class="container__date"><?php the_time( 'Y/m/d' ); ?></p>
+	<?php if ( has_post_thumbnail() ): ?>
+		<img class="container__thumbnail" src="<?= the_post_thumbnail_url(); ?>" alt="thumbnail">
+	<?php else: ?>
+		<img class="container__thumbnail" src="<?= esc_url( get_stylesheet_directory_uri() . '/img/dummy.jpg' ); ?>" alt="dummy-thumbnail">
+	<?php endif; ?>
+	<p class="container__contents"><?php the_field( 'contents' ); ?></p>
+</article>
+<u><a class="back-link" href="<?= get_post_type_archive_link( 'works' ); ?>">一覧に戻る</a></u>
```

Comment:
```suggestion
<a class="back-link" href="<?= get_post_type_archive_link( 'works' ); ?>">一覧に戻る</a>
```
uタグはhtml5以降非推奨っぽいのでやめましょう。
アンダーラインなどの見た目の制御はhtmlでどうにかするのではなくcssで操作するのがセオリーです。

Reply:
done https://github.com/oooshima/origin_wordpress/pull/28/commits/22a25f862e5cadaba4bf52cbe987491dc2d86b04